File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromInternalToProductionTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.GooglePlayPublisher;

public class PromoteFromInternalToProductionTask extends BaseTask {
	
	public PromoteFromInternalToProductionTask() {
		setDescription("Promote a current internal to production");
	}

	@Override
	protected void onExecute(App app) {
		GooglePlayPublisher.promoteFromInternalToProduction(app);
	}

}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromInternalToProductionTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

open class PromoteFromInternalToProductionTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().promoteFromInternalToProduction(app)
    }

    init {
        description = "Promote a current internal to production"
    }
}
